deployer/handlers.go

```diff
@@ -32,7 +33,7 @@ func Validate(awsc aws.Clients) DeployHandler {
 		release.SetDefaults(region, account)
 
 		if err := release.Validate(awsc.S3(release.AwsRegion, nil, nil)); err != nil {
-			return nil, &errors.BadReleaseError{err.Error()}
+			return nil, &errors.BadReleaseError{Cause: err.Error()}
```

Comment:
Equivalent, but explicit I assume?

Reply:
Yep, and safer in case the layout of `BadReleaseError` ever changes. Required by `go vet`.